src/pages/patientView/PatientViewPage.tsx

```diff
@@ -247,6 +259,61 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
         return this.tsInternalClient.getSignificantlyMutatedGenesUsingGET({studyId: this.studyId});
     }
 
+    fetchVariantCountData(mutations:Mutation[]):Promise<VariantCountData> {
+        return new Promise((resolve, reject) => {
+            const variantCountIdentifiers:VariantCountIdentifier[] = [];
+            const toQuery:{ [entrezGeneId:string]:{ [keyword:string]:boolean}} = {};
+            for (let mutation of mutations) {
```

Comment:
I only used it here because I thought that it does change by being reset during the iteration